src/python/WMCore/WMSpec/StdSpecs/TaskChain.py

```diff
@@ -400,9 +399,7 @@ def setupTask(self, task, taskConf):
                                               splitArgs = taskConf["SplittingArguments"],
                                               stepType = cmsswStepType,
                                               forceUnmerged = forceUnmerged,
-                                              timePerEvent = self.timePerEvent,
                                               memoryReq = taskConf.get("Memory", None),
-                                              sizePerEvent = self.sizePerEvent, 
```

Comment:
Ok, finally got it! One thing that confused me is that we were passing the timePerEvent and sizePerEvent from the spec (instead of the one set in the task). The current code in production seems to be working fine, but weren't we supposed to pass
timePerEvent = taskConf.get('TimPerEvent', None)
instead? The same for sizePerEvent.